### ISSUE TITLE
NTR: 3-hydroxyisobutyric acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8609]
New term request 3-hydroxyisobutyric acid high in urine (segal_030513172037):   UMLS: 4229730

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000607
- label: Elevated urinary 3-hydroxyisobutyric acid level
- parents: [Abnormal urine carboxylic acid level (HP:0031980)](https://hpo.jax.org/app/browse/term/HP:0031980)
- pmid: PMID:23835272
- def: The amount of 3-hydroxyisobutyric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8609
- synonyms:
